README.md

```diff
@@ -29,3 +29,31 @@ Gitpod is provided as a [managed Saas version](https://gitpod.io) with a free su
 Open any GitLab, GitHub, or Bitbucket project, by prefixing the project's URL with `gitpod.io/#`. 
 
 E.g. https://gitpod.io/#https://github.com/gitpod-io/gitpod
```

Comment:
**thought**: We could add a more expanded getting started section, following the update in the [getting started section in docs](https://www.gitpod.io/docs/getting-started/).

```suggestion
You can start using Gitpod with one or more of the following ways:

1. [Use a Prefixed URL](https://www.gitpod.io/docs/context-urls/)
1. [Install Browser Extension](https://www.gitpod.io/docs/browser-extension/)
1. [Enable GitLab Integration](https://www.gitpod.io/docs/gitlab-integration/)
1. Quick start using an [Example Project](https://www.gitpod.io/docs/getting-started/#example-project) or [OSS Project](https://www.gitpod.io/docs/getting-started/#gitpodified-open-source-project)
```